lib/grammers-client/src/types/callback_query.rs

```diff
@@ -142,11 +172,19 @@ impl<'a> Answer<'a> {
     pub async fn edit<M: Into<InputMessage>>(self, new_message: M) -> Result<(), InvocationError> {
         self.query.client.invoke(&self.request).await?;
         let chat = self.query.chat();
-        let msg_id = self.query.raw.msg_id;
-        self.query
-            .client
-            .edit_message(chat, msg_id, new_message)
-            .await
+        if let Some(ref msg_id) = self.query.inline_msg_id {
+            self.query
+                .client
+                .edit_inline_message(msg_id.clone(), new_message)
+                .await
+                .map(|_| ())
```

Comment:
IIRC `map(drop)` was preferred as this looks like a toilet.

Reply:
kkkk, sure